inc/apirest.class.php

```diff
@@ -446,6 +446,11 @@ public function parseIncomingParams($is_inline_doc = false) {
       if (function_exists('getallheaders')) {
          //apache specific
          $headers = getallheaders();
+         if (false !== $headers) {
```

Comment:
`count($headers)` could be a better test

Reply:
I added both, the function returns false on some cases.